indexer/tests/test_importer.py

```diff
@@ -166,9 +166,54 @@ class TestElasticsearchImporter:
     @pytest.fixture
     def importer(self) -> ElasticsearchImporter:
         importer = ElasticsearchImporter("test_importer", "elasticsearch import worker")
+        importer.elasticsearch_hosts = os.environ.get("ELASTICSEARCH_HOSTS")
         return importer
 
     def test_import_story_success(self, importer: ElasticsearchImporter) -> None:
         test_import_data = {**test_data, "url": "http://example_import_story.com"}
         assert importer.import_story(test_import_data)
+
+    def test_import_story_extra_fields(self, importer: ElasticsearchImporter) -> None:
+        test_import_data = {
+            **test_data,
+            "url": "https://damienafsoe.ttblogs.com/4775282/the-basic-principles-of-would-or-could",
+            "normalized_article_title": "the basic principles of would or could",
+            "normalized_url": "http://damienafsoe.ttblogs.com/4775282/the-basic-principles-of-would-or-could",
+            "text_extraction_method": "trafilatura",
+        }
```

Comment:
```suggestion
        test_extra_data = {
            "normalized_article_title": "the basic principles of would or could",
            "normalized_url": "http://damienafsoe.ttblogs.com/4775282/the-basic-principles-of-would-or-could",
            "text_extraction_method": "trafilatura",
        }
        test_import_data = {
            **test_data,
            **test_extra_data,
            "url": "https://damienafsoe.ttblogs.com/4775282/the-basic-principles-of-would-or-could",
        }
```

We can then use `test_extra_data` in the final asset without having to loop/dictionary comprehension